audits/bravo.py

```diff
@@ -233,19 +238,20 @@ def get_sample_sizes(self, contests, margins, sample_results):
 
         samples = {}
 
-        asns = self.get_expected_sample_sizes(margins, contests, sample_results)
+        asns = self.get_expected_sample_sizes(margins, contests,
+                                              sample_results)
         for contest in contests:
             samples[contest] = {}
 
             p_w = 10**7
-            s_w = 0 
+            s_w = 0
             p_l = 0
             best_loser = ''
             worse_winner = ''
 
-
             # For multi-winner, do nothing
-            if 'numWinners' not in contests[contest] or contests[contest]['numWinners'] != 1:
+            if 'numWinners' not in contests[
+                    contest] or contests[contest]['numWinners'] != 1:
```

Comment:
Here's another instance of it splitting on braces, it kind of makes the code hard to read.